### PR TITLE
Use positive rpm in shooter

### PR DESCRIPTION
**Pull request by dwillins**

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/Shooter.java [hunk 1]
```diff
@@ -216,7 +217,7 @@ public double calculateShooterHood(double distance) {
     }
 
     public void warmUp () {
-        pidController.setReference(Constants.WARM_UP_RPM, ControlType.kVelocity);
+        setShooterRpm(3000);
```
Comment (prensing) [inline src/main/java/frc/robot/subsystems/Shooter.java:220]: Hmm, why not use the Constant?

**Review by prensing — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/subsystems/Shooter.java (hunk 1 above)
Comment (dwillins) [reply src/main/java/frc/robot/subsystems/Shooter.java]: not really sure what I was thinking there, I just changed it to use the constant

File: src/main/java/frc/robot/subsystems/Shooter.java [hunk 2]
```diff
@@ -167,13 +167,14 @@ public void shoot () {
     }
 
     public void testSpin () {
-        pidController.setReference(-4000, ControlType.kVelocity);
+        setShooterRpm(4000.0);
         SmartDashboard.putString("Shooting", "Shooting");
     }
 
-    public void setShooterRPM (double rpm) {
+    public void setShooterRpm (double rpm) {
```
Comment (coachFitz0807) [inline src/main/java/frc/robot/subsystems/Shooter.java:174]: Please add comments before the Line 174 and/or before Line 176 explaining why we are negating the input rpm value.

**Review by coachFitz0807 — CHANGES_REQUESTED** [review]

**Review by prensing — APPROVED** [review]
Looks good to me.

**Review by coachFitz0807 — APPROVED** [review]

**Review by coachFitz0807 — APPROVED** [review]

**Pull request merged**